src/change/sdp.rs

```diff
@@ -800,7 +800,9 @@ fn add_new_lines(
             media.need_open_event = need_open_event;
 
             // Match/narrow remote params.
-            session.codec_config.update_params(&m.rtp_params());
+            session
+                .codec_config
+                .update_params(&m.rtp_params(), m.direction());
```

Comment:
The direction needs to be inverted here since `Media::direction` returns the local direction. Same in `update_media`

Reply:
This is m (remote m-line). Not media.

Reply:
Ah, so it is